doc/releases/changelog-dev.md

```diff
@@ -112,6 +112,9 @@
 * `qml.load` has been removed in favour of more specific functions, such as `qml.from_qiskit`, etc.
   [(#5654)](https://github.com/PennyLaneAI/pennylane/pull/5654)
 
+* `qml.TrotterProduct` is now compatible with resource tracking by inheriting from `ResourcesOperation`. 
+   [(#5680)](https://github.com/PennyLaneAI/pennylane/pull/5680)
+
```

Comment:
Why it is under Breaking changes? Is there a bug report for this?

Reply:
Ahh nice catch, let me move it to improvements

Reply:
Moved !